extensions/amp-addthis/0.1/amp-addthis.js

```diff
@@ -223,6 +223,25 @@ class AmpAddThis extends AMP.BaseElement {
       // Only the component that registers the page view listens for x-frame
       // events.
       this.setupListeners_({ampDoc, loc, pubId: this.pubId_});
+
+      // Create close button for listing tool
+      if (this.element.getAttribute('data-widget-type') === 'messages') {
+        const closeButton = createElementWithAttributes(
+          this.win.document,
+          'div',
+          dict({
+            'id': 'closeButton',
```

Comment:
Use an element-specific class instead of `[id]`, e.g. `.i-amphtml-addthis-close`.